### DISCUSSION TITLE
皮肤站转移后图片丢失

### DISCUSSION THREAD

**Discussion by ZHI-CCC** [bs-community/blessing-skin-server#561] [Q&A]
### Blessing Skin 版本

	6.0.2

### PHP 版本

8.0

### 出现问题时所使用的浏览器

Microsoft Edge

### 您正在使用的 Web Server

Nginx

### 您正在使用宝塔吗？

- [X] 是

### 出现了什么问题？

换机子后,手动打包源文件然后转移
就出现了这种情况
![image](https://user-images.githubusercontent.com/123163148/218262687-a0c651be-0835-4122-87d1-dd4cb350b8aa.png)
所有的预览图丢失
并且头像图片也丢失

### 错误日志

```text
无
```

### 重现步骤

转移了机子,新安装的面板环境
数据库原封不动的转移了过去

**Comment by Jerry12228** [comment]
我赌你PHP的imagemagick扩展没装
如果装了，就打开debug，然后f12找到缺失图片的地址，浏览器访问看报什么错（一种方法，可能不够专业，将就用）
